docs/user_guide.rst

```diff
@@ -1,2 +1,44 @@
 User Guide
 ==========
+
+.. contents:: :depth: 3
+
+:Note: FoxPuppet uses Selenium as it's main driver. That means Selenium commands work
+       right along with FoxPuppet specific commands.
+
+Example
+-------
+
+This is an example for setting up FoxPuppet and running a simple check to see if the browser window is private::
+
+    from foxpuppet import FoxPuppet
+    from selenium.webdriver import Firefox
+
+
+    selenium = Firefox
+    foxpuppet = FoxPuppet(selenium)
+
+    assert not foxpuppet.browser.is_private
+    foxpuppet.browser.open_window(private=True)
+    assert foxpuppet.browser.is_private
```

Comment:
Don't include asserts in your examples, as this presumes the user is writing a test. I would suggest opening a new window, and then closing it. We can add more to this example when we have more functionality. It would be neat to do something like type an address into the awesome bar, and then save it as a bookmark.